new-readme.md

```diff
@@ -3,30 +3,31 @@
 
 [![Build Status](https://travis-ci.org/reimagined/resolve.svg?branch=master)](https://travis-ci.org/reimagined/resolve) [![npm version](https://badge.fury.io/js/create-resolve-app.svg)](https://badge.fury.io/js/create-resolve-app) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/reimagined/resolve/master/LICENSE)
 
-reSolve is a framework for developing apps based on **CQRS**, **Domain-Driven Design** and **Event Sourcing** using **React + Redux** on the client. It negates differences between your domain and technical languages, and allows you to focus on application code.
+A JavaScript framework for **CQRS**, **Domain-Driven Design** and **Event Sourcing** with good support for **React** and **Redux**. **reSolve** provides a new way to describe app's core logic and fully separate it from an auxiliary code.
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-technologies.png"></p>
 
-* **2 minute** easy [setup](#installation)
-* **Free and open source.** reSolve is free to use under the [MIT license](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
-* **Flexibility.** Great opportunities with [clear API](#documentation).
+* The [Installation](#installation) is fully automated and takes 2 minutes.
+* [Free and open source](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
+* Flexible and clear [API](#documentation).
 
+<!-- TODO: text="Try it now!" -->
```

Comment:
We have a button 'getting started'. Yesterday we received a good beautiful button from designer.

Reply:
Good, but It should be at least "Get started"

Reply:
https://www.screencast.com/t/R9pAEntl

If it's wrong we'll fix it and remade design. Wrong?

Reply:
As I wrote, the text should be "Try now!" or "Try it now!"